backend/server.ts

```diff
@@ -0,0 +1,62 @@
+import { executeQuery } from "./psql/queryExecute";
+import { QUERY_GET_ALL_DAOS } from "./psql/query";
+const express = require("express");
```

Comment:
Shoud we change code to use `import..from` ?

Reply:
No, it needs require.